src/lib/shady-render.ts

```diff
@@ -131,7 +131,7 @@ const prepareTemplateStyles =
       // Template's underlying template element. Otherwise, we create one here
       // to give to ShadyCSS, which still requires one while scoping.
       const templateElement =
-          !!template ? template.element : document.createElement('template');
```

Comment:
We prefer only using booleans in conditionals

Reply:
Recommendation and auto-fix from https://eslint.org/docs/rules/no-extra-boolean-cast